src/common/http.ts

```diff
@@ -44,16 +43,6 @@ export class TurboHTTPService implements TurboHTTPServiceInterface {
       axiosConfig: {
         baseURL: url,
         maxRedirects: 0, // prevents backpressure issues when uploading larger streams via https
-        onUploadProgress: (progressEvent) => {
-          this.logger.debug(`Uploading...`, {
-            percent: Math.floor((progressEvent.progress ?? 0) * 100),
-            loaded: `${progressEvent.loaded} bytes`,
-            total: `${progressEvent.total} bytes`,
-          });
-          if (progressEvent.progress === 1) {
-            this.logger.debug(`Upload complete!`);
-          }
-        },
```

Comment:
while this is useful for actual bytes moving through axios, it overloads our upload progress logs that we attach to to the stream being uploaded so removing it